The minimum Maven version is to low to build the Manual subproject.

Karaf 4.4.9
The minimum Maven Version, per the enforcer plugin, is 3.3.9 is to low for the asciidoctor-maven-plugin used by Karaf's Manual sub-project.  asciidoctor-maven-plugin v3.2.0 has a minimum maven version of 3.8.8.

This was discovered while building Karaf on Debian Bookworm, which include Maven 3.8.7.

> [ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:3.2.0:process-asciidoc (output-html) on project manual: The plugin org.asciidoctor:asciidoctor-maven-plugin:3.2.0 requires Maven version 3.8.8 -> [Help 1]

--- a/pom.xml
+++ b/pom.xml
@@ -625,10 +625,10 @@
                         <configuration>
                             <rules>
                                 <requireMavenVersion>
-                                    <version>[3.3.9,4)</version>
+                                    <version>[3.8.8,4)</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>[11,)</version>
+                                    <version>[17,)</version>
                                 </requireJavaVersion>
                             </rules>
                         </configuration>

